javascript/packages/orchestrator/src/types.ts

```diff
@@ -49,6 +49,7 @@ export interface RelayChainConfig {
   chain_spec_command?: string;
   default_args?: string[];
   default_overrides?: Override[];
+  chain_spec_modifier_commands?: string[][];
```

Comment:
How about this: 

In types.ts:
```ts
type Command = string[];
```

```suggestion
  chain_spec_modifier_commands?: Command[];
```